.github/workflows/publish.yml

```diff
@@ -15,7 +15,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - name: Set up Homebrew
-        uses: Homebrew/actions/setup-homebrew@master
+        # NOTE: A specific commit is used here b/c this action does not
+        #       publish tags and the current `master` appears to be broken.
+        uses: Homebrew/actions/setup-homebrew@449449e64aaa5faac0d24229a2ffba32438e4b63
```

Comment:
Can we try master again now that Louis' system is updated?

Reply:
Yes. It was attempted in #20, but failed for a new/different reason. The recommended approach now is to accept this (#19) PR and move on...perhaps with an issue filed in the `Homebrew/actions` repository.